test/blackbox-tests/test-cases/dune-project-meta/proj

```diff
@@ -0,0 +1,31 @@
+(lang dune 1.10)
+(name cohttp)
+(source (github mirage/ocaml-cohttp))
```

Comment:
Just a thought here: we could shorten this to the common case of

```
(source mirage/ocaml-cohttp)
```

to make the `(github mirage/ocaml-cohttp)` the default.  Can the dune parser do the magic default if just an Atom is specified instead of a Sexp field?

Reply:
Yes it can. Is it really necessary however? I like github but perhaps we should keep support open for other hosting providers

Reply:
Not suggesting removing support for other providers; just making the common case shorter syntax default to github (as e.g. the brew CLI does). I’m also fine with leaving as is